src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -2041,21 +1886,50 @@ public static StackValue PerformReturnTypeCoerce(FunctionEndPoint functionEndPoi
         #endregion
 
         /// <summary>
-        /// Returns a flat collection of ISerializable objects from a serialized representation of a SingleRunTraceData object.
+        /// Returns a flat collection of strings from a serialized representation of a SingleRunTraceData object.
         /// </summary>
         /// <param name="callSiteData">The serialized representation of a SingleRunTraceData object.</param>
-        /// <returns>A flat collection of ISerializable objects.</returns>
-        public static IList<ISerializable> GetAllSerializablesFromSingleRunTraceData(
+        /// <returns>A flat collection of strings.</returns>
+        public static IList<string> GetAllSerializablesFromSingleRunTraceData(
```

Comment:
On thing we may be able to verify is does this need to be public.  Internal it is only called one place `GetOrphanedSerializablesAndClearHistoricalTraceData`...  Maybe in terms of quick validation of old data we can do it upstream and escape calling this one by one per node guid.

Reply:
Hmmm. Looks like D4C3D calls it in their own version of `GetAllSerializablesFromSingleRunTraceData`.

Reply:
So maybe stays public but we can also escape calls upstream and discuss with D4C3D the same optimization